client/src/layouts/RegistrationLayout/index.tsx

```diff
@@ -0,0 +1,20 @@
+import clsx from "clsx";
```

Comment:
I think we should reduce the number of packages to one for classNames - `clsx`, `classnames`